### PR TITLE
change container to service

### PR DESCRIPTION
**Pull request by Salem-Luos**

### CODE REVIEW

**Review by Simonbdy — APPROVED** [review]
I fixed a few "service" errors (one constant and a few classes in CSS files), and corrected typos from previous comits. All the rest is good.

However, I noticed a few changes in code blocks (functions, variables, constant, etc.) with "service": 
service_t
service_stats_t
Luos_CreateService()
Service_MsgHandler()
RoutingTB_DetectServices()
RoutingTB_IDFromService()
RoutingTB_ServiceIsSensor()
RoutingTB_ConvertServiceToRoutingTable()
RoutingTB_GetLastService()
TemplateState_CreateService()
MAX_SERVICE_NUMBER
service_alias
etc.

I haven't touched any of these (except from updating the uppercases in one constant), so if the code parts are ok for you, you can merge.

**Review by Simonbdy — CHANGES_REQUESTED** [review]
I made a few comments on rephrasing sentences and improving images' visuals.

File: src/pages/luos-technology/node/topology.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Network topology
+
+As mentioned in the previous paragraphs, thanks to [Robus](../node/luos.md), the nodes have the capabilty to communicate with each other. However, Robus is not the only necessity to achieve using this characteristic. The nodes can have at any time, knowledge about the position of every other node in the network, by accessing a specific structure, called Routing Table. The routing table, is constructed and shared among the nodes, after the execution of a devoted process called detection.
```
[the remark this answers is not in the document]
Comment (BiliouriV) [reply src/pages/luos-technology/node/topology.md]: I believe that you can totally delete this sentence. I used it as a transition to introduce the idea of topology detection, but you are right it is not clear.  I think that it is not necessary

**Review by nicolas-rabault — COMMENTED** [review]
I apply my modifications.

**Review by Simonbdy — COMMENTED** [review]
@nicolas-rabault could you review both new FAQ pages from Thomas?

**Comment by Simonbdy** [comment]
New comments:
1. In the topology boxes drawn with box-drawing characters, the right stroke is shifted on these pages: pyluos.md, get_started.md, . @nicolas-rabault I think you made these changes, can you please check and fix them?
2.  Various images have various shapes and colors, I will open an issue so that the same codes are used for every image in the docs

**Comment by Simonbdy** [comment]
I had been able to fix the box-drawing char problem (1) by removing "Source Code Pro" from the `font-family` CSS for the `<code>` tag in file `general.css`.

File: src/pages/luos-technology/basics/archi.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+    **Package.c/.h**: Contains the Service_Init, Service_Loop, and Service_Msg_Handler. In these files, you can create, with luos API, services dedicated to your function
+
+- **Service level**
+    Use luos API and Service Profile to create a standard or a custom service that can be used by anybody
+
+### Where is Luos?
+
+Luos Library is used at the node level.
+
+![](../../../_assets/img/luos_mcu_platform.png)
```
[the remark this answers is not in the document]
Comment (Simonbdy) [reply src/pages/luos-technology/basics/archi.md]: Addressed in #127 for the next version

**Review by nicolas-rabault — APPROVED** [review]
Realy nice.

**Review by Simonbdy — APPROVED** [review]
Good job!

**Pull request merged**